uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-entitlements.html

```diff
@@ -1 +1,154 @@
-Entitlements
+<section class="panel panel-default panel-default-plain">
+  <header class="panel-heading">
+    <h2 class="panel-title">Contract Entitlements</h2>
+    <hr/>
+  </header>
+  <div class="table-responsive">
```

Comment:
Here it is suggested to use a wrapping div http://getbootstrap.com/css/#tables-responsive. Moreover, `table-responsive` class doesn't seem to work if specified inside a table node.